src/components/NetworkSwitcher.tsx

```diff
@@ -595,34 +648,9 @@ function NetworksGrid({
       dragging.value = null;
     });
 
-  const pressedExpand = useSharedValue(false);
-
-  // TODO: Need to prevent this from firing the tapNetwork as well
-  const tapExpand = Gesture.Tap()
-    .onBegin(e => {
-      if (editing.value) {
-        e.state = State.FAILED;
-      }
-      pressedExpand.value = true;
-    })
-    .onEnd(() => {
-      pressedExpand.value = false;
-      expanded.value = !expanded.value;
-    });
-
   const tapNetwork = Gesture.Tap()
     .onTouchesDown((e, s) => {
-      if (editing.value) {
-        s.fail();
-      }
-
-      const touches = e.allTouches[0];
-      const section = touches.y > sectionsOffsets.value[Section.unpinned].y ? Section.unpinned : Section.pinned;
-      const index = indexFromPosition(touches.x, touches.y, sectionsOffsets.value[section]);
-      const chainId = networks.value[section][index];
-      if (!chainId) {
-        s.fail();
-      }
```

Comment:
why was this removed? This previously would fail early if you tapped an empty column

Reply:
we were doing the same work on onTouchesDown and onEnd, doesn't make sense to fail early if we did the full work

Reply:
can you verify that this doesn't allow clicking in a blank spot and messing up the UI? IIRC there was definitely a reason I added this here but could be wrong

Reply:
seems to work fine